test/util/genesis/accounts.go

```diff
@@ -15,34 +16,41 @@ import (
 	"github.com/tendermint/tendermint/crypto"
 )
 
-type Account struct {
+const (
+	DefaultInitialBalance = 1e15 // 1 billion TIA
+)
+
+// KeyringAccount represents a user account on the Celestia network.
+// Either the name, if using the genesis keyring, or an address
+// needs to be provided
```

Comment:
Needs to be provided where? Perhaps in the constructor but if so then this comment should live above the constructor.

```suggestion
```

Reply:
I took genesis from main